control_frame: gps_garmin_origin
N: 2
UAV_names: [uav3, uav7]
clearance: 2.5
clearance_obstacles: 0.5
rho_act: 10.0
pot_strength: 1000.0
r : 0.4
m_soft : 1.5 
m_hard : 3
ros_rate : 10 
ts : 0.01
Kv : 0.25
lambda: 20
v_max: 2.0
c_obst_x: [-35.0, -15.0, -25.0]
c_obst_y: [40, 30, 70]
r_obst: [2.0, 2.0, 2.0]
